modules/libcom/src/osi/os/RTEMS-score/osdEvent.c

```diff
@@ -48,11 +49,9 @@ epicsEventCreate(epicsEventInitialState initialState)
     rtems_status_code sc;
     rtems_id sid;
     rtems_interrupt_level level;
-    static char c1 = 'a';
-    static char c2 = 'a';
-    static char c3 = 'a';
+    static uint32_t name;
```

Comment:
Is using static here thread safe?

Reply:
No.  Neither the new nor is the current c1,c2,c3 implementation.

My understanding is that duplicate names would be confusing, but not a functional issue.  So I wasn't going to bother with the "old" RTEMS series.

Reply:
Actually, scratch this.  Both implementations are guarded by `rtems_interrupt_enable()`.  This PR move that logic into `next_rtems_name ()` to avoid duplication.

Reply:
OK